### PR TITLE
Use data-jupyter-widgets-cdn-only attribute to load modules only from CDN

### PR DESCRIPTION
**Pull request by Bobgy**
Resolves https://github.com/jupyter-widgets/ipywidgets/issues/2786

This PR's changes are learned from previous similar PR: https://github.com/jupyter-widgets/ipywidgets/commit/8aba1831fca8787d31761539e224fc6faf19319b

Hi @jasongrout!
Can you help me review this?

I'm not sure how to test, because if I run `yarn test`, this fails at build step already in master branch.

### CODE REVIEW

**Comment by Bobgy** [comment]
@jasongrout Friendly ping

**Comment by Bobgy** [comment]
@jasongrout Would you have time to take a look?

File: packages/html-manager/src/libembed-amd.ts [hunk 1]
```diff
@@ -4,11 +4,14 @@
...
 
 let cdn = 'https://unpkg.com/';
+let onlyCDN = false;
 
 // find the data-cdn for any script tag, assuming it is only used for embed-amd.js
 const scripts = document.getElementsByTagName('script');
 Array.prototype.forEach.call(scripts, (script: HTMLScriptElement) => {
   cdn = script.getAttribute('data-jupyter-widgets-cdn') || cdn;
+  onlyCDN =
+    onlyCDN || script.getAttribute('data-jupyter-widgets-cdn-only') != null;
```
Comment (jasongrout) [inline packages/html-manager/src/libembed-amd.ts]: Can you use `hasAttribute` instead? https://developer.mozilla.org/en-US/docs/Web/API/Element/hasAttribute

Comment (Bobgy) [reply packages/html-manager/src/libembed-amd.ts]: TIL, Thanks!

**Comment by jasongrout** [comment]
Thanks! This looks okay to me. Congratulations on your first contribution to ipywidgets!

**Comment by Bobgy** [comment]
Thanks @jasongrout for the review!

**Pull request merged**